### ISSUE TITLE
remmina-1.2 SSH support in Fedora-24beta totally broken

### ISSUE THREAD

**Issue by jhaar** [FreeRDP/Remmina#899]
Please give us some details about the systems you are using:
* Client (OS name and version): Fedora-24beta
* Remmina version (remmina --version): remmina-1.2.0-0.13.rcgit.7.fc24.x86_64
* Desktop environment (GNOME, Unity, KDE, ..): gnome-shell-3.20.2-1.fc24.x86_64
* Connecting to (OS and version): any SSH
* Connecting via (RDP, VNC, ...): SSH

Hi there, I have already reported this to Redhat (#1340331), but if this is remmina related, I thought this might be faster ;-)

Anyway, their remmina-1.2.0-0.13.rcgit.7.fc24.x86_64 build basically doesn't work at all for SSH sessions. If I try a "Quick Connect", I get prompted for the password, and then it hangs - and the remmina process takes all the CPU. Similarly, if I create a pre-shared key SSH profile, same thing - hangs, never logs in and the fan comes on :-)

Weirdly enough, RDP sessions work fine - just not SSH. Obviously SSH works fine from terminals and vinagre - just not remmina

I don't know if this is some Fedora-24 library dependency bug, or some library update that triggers a bug in remmina - but I'm hoping you might be able to tell the difference?

Thanks! 

Jason

**Comment by antenore** [comment]
Thanks for the report.

This is something that we fixed when VTE has been upgraded, so it's weird that it doesn't work with Fedora 24... 

Can you please provide the vte library version?

I wont be able to look at it very soon.

**Comment by jhaar** [comment]
I have the following "vte" packages installed

vte3-0.36.5-2.fc24.x86_64
vte291-0.44.2-1.fc24.x86_64

I can also see there is a "vte-0.28.2-17.fc24" available to install, remmina itself appears linked to /lib64/libvte-2.91.so.0

**Comment by antenore** [comment]
It'd be the right version. I need a test environment and it won't happen very soon. :-\

**Comment by piotrantosik** [comment]
same problem but update to latest build (http://koji.fedoraproject.org/koji/packageinfo?packageID=10031 remmina-1.2.0-0.18.rcgit.14.fc25) fix this.

**Comment by antenore** [comment]
Reported by @Dolphykins (please update to last version)

> After upgrading a fairly fresh install of Fedora 23 Workstation to Fedora 24, SSH sessions (saved and new) are launching a blank window with a blinking cursor for me. This procedure was followed for the OS upgrade: https://fedoramagazine.org/upgrading-fedora-23-workstation-to-fedora-24/
> 
> Client: Fedora 24, Kernel 4.5.7-300.fc24.x86_64
> Remmina Version 1.2.0, Release 0.13.rcgit.7.fc24
> Gnome Shell 3.20.2
> Connecting to: FreeBSD 7.x-10.x, Kubuntu 15.10
> Connecting via: SSH
> 
> SSH connections work fine in Gnome-terminal, attached a screenshot of the SSH window after connecting and entering credentials.
> 
> ![screenshot from 2016-06-23 14-01-02](https://cloud.githubusercontent.com/assets/20116277/16318161/17facff2-394b-11e6-8232-831bad49e55f.png)
> 
>

**Comment by antenore** [comment]
@jhaar @Dolphykins Upgrade to last remmina version and let me know.

**Comment by Dolphykins** [comment]
@antenore Thanks for the fix, guys. Did an in-place install of the x86_64 rpm for remmina-1.2.0-0.19.rcgit.14.fc25 and SSH connections are working again.

**Comment by antenore** [comment]
I love when I do nothing and problems goes away :-)
I close this.

**Comment by Hubbitus** [comment]
Hi @antenore. If I understand correctly you fix this issue? I do not see any linked commit to try backport it. Is fix simple to escalate?

**Comment by antenore** [comment]
I did nothing, but it seems it works.

**Comment by Hubbitus** [comment]
@RangerGH, which plugin you can't install? What version of remmina you are use and from what source? Did you try version from copr repo remmina-next?

**Comment by RangerGH** [comment]
Once I installed the newer remmina can't install any plugin because they are 1.13 versions so that's understandable.

As I recall I found a link on an rpmfind site and downloaded from there.  I then did a dnf install ./rpmfile.

I was not aware you had a copr repo at all.  I saw the Ubuntu instructions on the site and instructions for compiling for Fedora 20 but at this point I'd rather not have to compile it.

Is this the copr repo you are referring to?  If so I'll try it.

https://copr.fedorainfracloud.org/coprs/hubbitus/remmina-next/
